### PR TITLE
Adds objects and elements for InsurancePolicy

### PR DESCRIPTION
**Pull request by kevinkirkl3y**

### CODE REVIEW

**Comment by kevinkirkl3y** [comment]
@cwhanse @jrippingale Here we go! Lets get this in before there are more merge conflicts.

**Review by cwhanse — APPROVED** [review]

File: Master-OB-OpenAPI.json [hunk 1]
```diff
@@ -11133,6 +11133,145 @@
...
+        ]
+      },
+      "InsurancePolicyID": {
+        "allOf": [
+          {
+            "$ref": "#/components/schemas/TaxonomyElementString"
+          },
+          {
+            "type": "object",
+            "description": "A unique identifier for a specific InsurancePolicy",
```
Comment (cwhanse) [inline Master-OB-OpenAPI.json:11198]:
```suggestion
            "description": "A unique identifier for a specific InsurancePolicy.",
```

File: Master-OB-OpenAPI.json [hunk 2]
```diff
@@ -11133,6 +11133,145 @@
...
+        ]
+      },
+      "AggregateAmount": {
+        "allOf": [
+          {
+            "$ref": "#/components/schemas/TaxonomyElementNumber"
+          },
+          {
+            "type": "object",
+            "description": "The highest amount of money the insurer will pay for all of your losses during a policy period.",
```
Comment (cwhanse) [inline Master-OB-OpenAPI.json:11162]:
```suggestion
            "description": "The maximum amount of money the insurer will pay for all of your losses during a policy period.",
```

File: Master-OB-OpenAPI.json [hunk 3]
```diff
@@ -16482,6 +16621,35 @@
...
+        "WorkersComp": {
+          "label": "Workers Compensation",
+          "description": ""
+        },
+        "LiabilityCasualty": {
+          "label": "Liability Casualty\t",
+          "description": ""
+        },
+        "SuretyBonds": {
+          "label": "Surety Bonds\t",
```
Comment (cwhanse) [inline Master-OB-OpenAPI.json:16637]:
```suggestion
          "label": "Surety Bonds",
```

File: Master-OB-OpenAPI.json [hunk 4]
```diff
@@ -16482,6 +16621,35 @@
...
+    },
+    "RiskTransferTypeItemType": {
+      "description": "Type of method or product that transfers risk between entities.",
+      "enums": {
+        "WorkersComp": {
+          "label": "Workers Compensation",
+          "description": ""
+        },
+        "LiabilityCasualty": {
+          "label": "Liability Casualty\t",
```
Comment (cwhanse) [inline Master-OB-OpenAPI.json:16633]:
```suggestion
          "label": "Liability Casualty",
```

**Review by cwhanse — APPROVED** [review]
some clean-up

**Pull request merged**